src/test/kotlin/com/terraformation/backend/customer/AppNotificationServiceTest.kt

```diff
@@ -263,7 +263,6 @@ internal class AppNotificationServiceTest : DatabaseTest(), RunsAsUser {
     every { user.canReadSpecies(any()) } returns true
     every { user.locale } returns Locale.ENGLISH
     every { user.organizationRoles } returns mapOf(organizationId to Role.Admin)
-    every { deliverableStore.fetchDeliverableSubmissions() } returns listOf(deliverable)
 
```

Comment:
In this test, unlike the email one, we're inserting real values into the database, so `deliverableStore` is a real instance of `DeliverableStore` rather than a mock.